### ISSUE TITLE
Question about results on ScanNet. Why result of paper v2 is significantly higher than that of paper v1?

### ISSUE THREAD

**Issue by charlesCXK** [facebookresearch/votenet#27]
Hi, I have a question about the results on ScanNet dataset. I find that mAP@0.25 is reported to be **46.75%** in the first version of your paper, while mAP@0.25 is boosted to be **58.6%** in the version 2. Could you tell me how do you gain such an amazing increase? Thank you !

**Comment by charlesCXK** [comment]
We reproduced VoteNet according to the setting in the paper versio-1, and got a reasonable result on SUNRGBD. However, we only got 49.8@0.25 on ScanNet, which is reasonable when compared with paper version-1, but is far lower than the result in paper version-2. Thus we create this issue.

**Comment by charlesq34** [comment]
Hi @charlesCXK 

The major change is the learning rate scheduling. Originally we have exponential decay of the learning rate for paper version-1. Later we updated it to step-wise learning rate decay with the same intervals as that for SUN RGB-D training (80,120,160 epochs). The delayed decay of the learning contributed to the performance boost.

**Comment by charlesCXK** [comment]
Thanks! We will check it. @charlesq34
